cypress/integration/my-orcid-sidebar.e2e.js

```diff
@@ -0,0 +1,119 @@
+describe('My Orcid sidebar' + runInfo(), () => {
+  describe('Orcid id', () => {
+    it('display the users Orcid', () => {})
+    it('display url to navigate to the public page view', () => {})
+  })
+  describe('Emails' + runInfo(), () => {
+    it('display a user with only a primary email "unverified"', () => {})
+    it('display a user with only a primary email "verified"', () => {})
+    it('add additional emails and display those as "unverified" with default privacy', () => {
+      // Expect changes to be display outside and inside of the modal
+      // TODO @dandineen no sketch that displays the input box
+    })
+    it('remove/delete  emails and display those as "unverified" with default privacy', () => {
+      // Expect changes to be display outside and inside of the modal
+    })
+    it('display emails "unverified" and "verified"', () => {
+      // Expect changes to be display outside and inside of the modal
+    })
+    it('resend the email verification email', () => {
+      // TODO @dandineen no button to resend
```

Comment:
I've added in a 'Verify email' link, sitting at the end of the field itself. Truth be told I'm not 100% sure about it. I may need to give the email form further thought to see if I can make something a little more elegant.

https://zpl.io/VYOlJqe